### DISCUSSION TITLE
Scoping styles to mobile + landscape doesn't seem to work?

### DISCUSSION THREAD

**Discussion by qodesmith** [tailwindlabs/tailwindcss#9950] [Help] [answered]
I am trying to apply styles to the mobile-only breakpoint (`max-md`) for the landscape orientation. It doesn't seem to work. Using a blue background color (`bg-blue-600`) to see the results, here's what I'm experiencing:

`max-md:landscape:bg-blue-600`
* Mobile - it correctly shows the blue background only when oriented in landscape ✅
* Desktop - it shows the blue background 😞

`landscape:max-md:bg-blue-600`
* Mobile - the blue background shows for both orientations 😞
* Desktop - the blue background does not show ✅

How can I get the blue background to show only at the mobile breakpoint when in the landscape orientation?

**Comment by wongjn** [comment, marked as answer]
I have just tested both classes in a [Tailwind Play](https://play.tailwindcss.com/LwTuOOzCCn?size=658x345) and they both seem to do what you'd like. Perhaps double-check whether the background color is coming from a different class. Otherwise, if you can provide a reproduction of your problem in a [Tailwind Play](https://play.tailwindcss.com/) or a remotely-hosted git repository, we can take a closer look.

**Reply by qodesmith** [reply]
Thanks for getting back so quick. You're right, it seems both cases work. I must be doing something wrong somewhere upstream, perhaps a conflicting style or something. I'll dig in deeper. Thank you!
